mteb/abstasks/languages.py

```diff
@@ -2,7 +2,10 @@
 Script codes (ISO 15924) obtained from: https://unicode.org/iso15924/iso15924.txt
 """
 
+from __future__ import annotations
```

Comment:
Why is this needed? We don't support Python 2, no?

Reply:
it is for the type annotation e.g. list[str] vs List[str]